openmetadata-docs/content/v1.4.x/connectors/database/oracle/index.md

```diff
@@ -40,20 +40,49 @@ CREATE ROLE new_role;
 -- GRANT ROLE TO USER 
 GRANT new_role TO user_name;
 
--- GRANT CREATE SESSION PRIVILEGE TO USER
+-- Grant CREATE SESSION Privilege.
+--   This allows the role to connect.
 GRANT CREATE SESSION TO new_role;
 
--- GRANT SELECT CATALOG ROLE PRIVILEGE TO FETCH METADATA TO ROLE / USER
+-- Grant SELECT_CATALOG_ROLE Privilege.
+--   This allows the role ReadOnly Access to Data Dictionaries
 GRANT SELECT_CATALOG_ROLE TO new_role;
 ```
 
+If you don't want to create a role, and directly give permissions to the user, you can take a look at an example given below.
+
+```sql
+-- Create a New User
+CREATE USER my_user IDENTIFIED by my_password;
+
+-- Grant CREATE SESSION Privilege.
+--   This allows the user to connect.
+GRANT CREATE SESSION TO my_user;
+
+-- Grant SELECT_CATALOG_ROLE Privilege.
+--   This allows the user ReadOnly Access to Data Dictionaries
+GRANT SELECT_CATALOG_ROLE to my_user;
+```
+
+
 With just these permissions, your user should be able to ingest the schemas, but not the tables inside them. To get
 the tables, you should grant `SELECT` permissions to the tables you are interested in. E.g.,
```

Comment:
is this still correct? With the SELECT_CATALOG_ROLE shouldn't the user be able to eat everything? cc @IceS2

Reply:
SELECT_CATALOG_ROLE is just for the dictionary & metadata access. To get them to profile and have DQ done on the table, the user will have to grant select on those tables explicitly

Reply:
it should be rather be, i will modify the line here
```
**Note**: With just these permissions, your user should be able to ingest the metadata, but not the `Profiler & Data Quality`, you should grant `SELECT` permissions to the tables you are interested in for the `Profiler & Data Quality` features to work.
```

Reply:
I have modified the statement there..Thanks.